src/lib/networks.config.ts

```diff
@@ -13,14 +13,10 @@ import { env } from '@/src/env'
 const devChains = [optimismSepolia, sepolia] as const
 const prodChains = [mainnet, polygon, arbitrum] as const
 const allChains = [...devChains, ...prodChains] as const
-
 export const chains = includeTestNets ? allChains : prodChains
```

Comment:
```suggestion
export const chains = includeTestnets ? allChains : prodChains
```
I'm pretty sure that the lowercase version is more commonly used.